_rules/meta-viewport-b4f0c3.md

```diff
@@ -83,6 +102,24 @@ This viewport `meta` element does not prevent user scaling because it has `user-
 
 #### Passed Example 2
 
+This viewport `meta` element does not prevent user scaling because it has `user-scalable` set to `5`.
+
+```html
+<html>
+	<head>
+		<title>Simple page showing random text</title>
+		<meta name="viewport" content="user-scalable=5" />
+	</head>
+	<body>
+		<p>
+			Lorem ipsum
+		</p>
+	</body>
+</html>
+```
```

Comment:
Is there a reason that this new example was added in the middle rather than the end? I'm not sure that we're strict about this, but if there's something out there already that refers to Pass Example 2, that reference would be bad since Pass Example 2 is now Pass Example 3. I'm sure it happens anyway when we remove examples, but I don't see a reason to do that in this case.

Reply:
Yes, I wanted to keep the `user-scalable` tests together, and the `maximum-scale` together. I think it makes the rale more readable than if we mix them (less need to switch context).

Our test cases numbers are absolutely not stable, and we should not link to them directly. This is acceptable when referring to them in issues, … but we shouldn't, e.g., have a rule linking to an example with a `passed-example-2` id. If we need to link inside rule, we should manually had an anchor.